parm/parm_fv3diag/diag_table_da

```diff
@@ -34,6 +34,13 @@
 "gfs_dyn",     "hs",          "hgtsfc",       "fv3_history",    "all",  .false.,  "none",  2
 
 "gfs_phys",    "cldfra",        "cldfra",        "fv3_history2d",  "all",  .false.,  "none",  2
+"gfs_phys",    "frzr",          "frzr",          "fv3_history2d",  "all",  .false.,  "none",  2
+"gfs_phys",    "frzrb",         "frzrb",         "fv3_history2d",  "all",  .false.,  "none",  2
+"gfs_phys",    "frozr",         "frozr",         "fv3_history2d",  "all",  .false.,  "none",  2
+"gfs_phys",    "frozrb",        "frozrb",        "fv3_history2d",  "all",  .false.,  "none",  2
+"gfs_phys",    "tsnowp",        "tsnowp",        "fv3_history2d",  "all",  .false.,  "none",  2
+"gfs_phys",    "tsnowpb",       "tsnowpb",       "fv3_history2d",  "all",  .false.,  "none",  2
+"gfs_phys",    "rhonewsn",      "rhonewsn",      "fv3_history2d",  "all",  .false.,  "none",  2
```

Comment:
Does the GSI read/need these? 
@CatherineThomas-NOAA @emilyhcliu 
I suggest we don't increase the volume of our output unless this is information needed by the DA executables.

Reply:
I wasn't sure myself, but forgot I wanted to ask DA about them when I opened the PR. Thanks Rahul.

Reply:
No, the GSI only reads the 3D hydrometeors.  

@jiaruidong2017 @CoryMartin-NOAA - Does the snow DA use these 2D variables at all?

Reply:
snow DA is currently using FMS restarts. These are the sfcfNNN history files, right? If so, snow DA is unaffected by this table.

Reply:
Thanks @CoryMartin-NOAA @CatherineThomas-NOAA 
We will remove adding these fields for now.
Please open an issue if you wish to add these back in to the surface Gaussian files for the GDAS cycle.

Reply:
Removed. Thanks everyone.